tests/integration/tier0/custom-kernel/install_custom_kernel.py

```diff
@@ -0,0 +1,41 @@
+import platform
+
+from envparse import env
+
+
+def test_install_custom_kernel(shell):
+    # We install CentOS kernel on Oracle Linux and vice versa to mimic the custom kernel
+    # that is not signed by the running OS official vendor
+    system_version = platform.platform()
+
+    if "centos-7" in system_version:
+        # Install dependency for kernel-uek
+        assert (
+            shell(
+                "yum install https://yum.oracle.com/repo/OracleLinux/OL7/latest/x86_64/getPackage/linux-firmware-20200124-999.4.git1eb2408c.el7.noarch.rpm -y"
+            ).returncode
+            == 0
+        )
+        assert (
+            shell(
+                "yum install https://yum.oracle.com/repo/OracleLinux/OL7/UEKR6/x86_64/getPackage/kernel-uek-5.4.17-2011.0.7.el7uek.x86_64.rpm -y"
+            ).returncode
+            == 0
```

Comment:
@kokesak does this still work? Because we have a check in Convert2RHEL to prevent UEK kernels to pass

Reply:
@r0x0d if the check works, then the conversion should inhibit. We can keep the UEK kernel install here as well as add a scenario where the non-uek kernel is installed on centos to test that as well.

Reply:
Right... I'm having some troubles to reproduce this locally because of those checks. Do you have a step-by-step on how to do it? 

Tried with the UEK and then the conversion halts on that check. Tried OL7 kernel, but it is the same version of CentOS 7 (no downgrades available for both of them)

Reply:
Hmm, maybe I had similar problem when installing OL7 kernel on CentOS so I used the UEK. I'll try it and let you know

Reply:
Okay, I tried to follow the steps in this test, but didn't work because of the checks we have in place for UEK

Reply:
@kokesak tried with this one and it worked: https://yum.oracle.com/repo/OracleLinux/OL7/latest/x86_64/getPackage/kernel-3.10.0-1160.76.1.0.1.el7.x86_64.rpm

Maybe we can adapt the test to include this kernel instead of the UEK

Reply:
Also, @kokesak, do you want to merge this PR with mine, so we get everything in one place? 

I can cherry-pick your changes here and merge into mine if you think that is worth it.

Reply:
@r0x0d nice that it works. It will be nice to have it under the same PR. I will add the commit to your PR myself and close this one, you dont have to do it.